examples/EntryToken/token.en.js

```diff
@@ -108,7 +108,7 @@ class Token {
         }
         return `<div>
             <div>
-                <span class="ts-count">x${this.props._count}</span>  <span class="ts-category">${this.props.name}</span>
+                <span class="ts-count">x${this.props._count}</span>  <span class="ts-category">${this.props.label}</span>
```

Comment:
@James-Sangalli this is the Token's "name" right (like "symbol" etc). Shouldn't be changed to "label"

Reply:
Yes but technically that is wrong according to the schema:

> <ts:label>
        <ts:string xml:lang="en">NEST</ts:string>
    </ts:label>

Reply:
@James-Sangalli Ouch. Ok, then have to sort it out with @colourful-land to see if we make `<ts:name>` here or  change make it `this.props.label`. (if the latter @James-Sangalli will have to undo the suggestion I made about the implicit var in the iOS codebase, so it would then be "label", sorry, James, or we can inject both).

The former seems more correct and less confusing to developers because its "token name", not "token label" in Ethereum-speak, smart contracts, ERC20, etc.

Reply:
> The former seems more correct and less confusing to developers because its "token name", not "token label" in Ethereum-speak, smart contracts, ERC20, etc.

I'm pretty sure when in Ethereum-speak, smart contract, ERC20, if you speak of token name, the developer won't be thinking FIFA门票 or Beilete de FIFA. They will be thinking stuff like "FIFA Ticket"